Framework/DataHandling/src/LoadNexusMonitors2.cpp

```diff
@@ -68,7 +68,8 @@ void loadSampleDataISIScompatibilityInfo(
 } // namespace
 
 //------------------------------------------------------------------------------
-LoadNexusMonitors2::LoadNexusMonitors2() : Algorithm(), m_monitor_count(0) {}
+LoadNexusMonitors2::LoadNexusMonitors2()
+    : Algorithm(), m_monitor_count(0), m_allMonitorsHaveHistoData(false) {}
```

Comment:
That would be nicer, but I kept this for consistency. I was thinking that a more project-wide revamp of initializations and constructors could be a maintenance task, either for this or the next maintenance period.